contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: "BitcoinUnlimited-win-0.12"
+name: "BitcoinUnlimited-win-1.0"
```

Comment:
I don't disagree, but it was important to provide the exact patch 1.0.1.2 was based upon for full disclosure after the release from the private repo and that includes a version change.  Version changes have not been part of any prior PR.